jujuclient/bootstrapconfigfile_test.go

```diff
@@ -79,6 +120,7 @@ var testBootstrapConfig = map[string]jujuclient.BootstrapConfig{
 		Cloud:               "maas",
 		CloudType:           "maas",
 		CloudRegion:         "127.0.0.1",
+		CloudCACertificates: []string{fakecert},
```

Comment:
Where are we checking that this cert is correctly added to the bootstrap config yaml?

Reply:
In environs/bootstrap/prepare_test.go, in TestPrepare, a cert is provided to the CloudSpec in the PrepareParams.  Then it checks the controllerStore, for the new controller, including an assert that the store's bootstrap config contains the cert.